PLAN.md

```diff
@@ -0,0 +1,57 @@
+# plan for opentelemetry-python-contrib
+This document captures the effort to move instrumentation and exporters out of the main repo and into the opentelemetry-python-contrib. Doing this will give us the ability to decouple changes to the core of OpenTelemetry (API/SDK) from changes to the rest of the eco-system. This pattern is utilized in other OpenTelemetry SIGs as well.
+
+## What lives where?
+The `opentelemetry-api`, `opentelemetry-sdk` and `opentelemetry-auto-instrumentation` packages will continue to live in the [opentelemetry-python](https://github.com/open-telemetry/opentelemetry-python) repo.
+
+The packages that currently exist in the [ext](https://github.com/open-telemetry/opentelemetry-python/tree/master/ext) folder will be migrated over to the [opentelemetry-python-contrib](https://github.com/open-telemetry/opentelemetry-python-contrib) repository. Currently, packages in that directory are all named `opentelemetry-ext-{package}`, in the contrib repo, these packages will be broken into `opentelemetry-instrumentation-{package}` and `opentelemetry-exporter-{package}`.
+
+## How does code end up in the contrib repo?
+There are currently two efforts in progress to bring code to life in the contrib repo.
+
+### migrating instrumentation/exporter from opentelemetry-python
+A lot of effort has gone into the all the code in the `ext` directory. In order not to lose that work, an effort has been started to migrate over the code to the new contrib directory.
```

Comment:
Is there a `contrib` directory?

Reply:
whoops, that should have said repo, will fix